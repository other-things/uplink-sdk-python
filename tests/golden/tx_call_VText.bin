00000000: 00 00 00 01 09 F9 55 4A  E9 80 4C 33 00 AB CA 52  ......UJ..L3...R
00000010: 3A 63 6A 53 AF 81 F6 39  AD 6E BA 17 0E 05 18 F0  :cjS...9.n......
00000020: 3F E2 9E 29 00 00 00 00  00 00 00 03 67 65 74 00  ?..)........get.
00000030: 00 00 00 00 00 00 01 07  00 0B 48 65 6C 6C 6F 20  ..........Hello 
00000040: 77 6F 72 6C 64                                    world