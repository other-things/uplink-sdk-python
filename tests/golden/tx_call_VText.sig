r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 99682820581079811605212233002762158228594299347414471242109551442404527669459
