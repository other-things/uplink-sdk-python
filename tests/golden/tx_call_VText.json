{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": "Hello world", 
                        "tag": "VText"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE05OTY4MjgyMDU4MTA3OTgxMTYwNTIxMjIzMzAwMjc2MjE1ODIyODU5NDI5OTM0NzQxNDQ3MTI0MjEwOTU1MTQ0MjQwNDUyNzY2OTQ1OQ=="
}